File: free_tasks/remove_duplicates.py
class Solution:
    list1 = [1,1,1,2,3,4,4,5]
    def removeDuplicates(list1):
        unique_list = []
  
    # traverse for all elements
        for x in list1:
        # check if exists in unique_list or not
            if x not in unique_list:
                unique_list.append(x)
    # print list
        return unique_list
    print(removeDuplicates(list1))

File: free_tasks/ft_js/remove_duplicates.js
class Solution {
    static removeDuplicates(list1) {
        const uniqueList = [];
        
        for (const x of list1) {
            if (!uniqueList.includes(x)) {
                uniqueList.push(x);
            }
        }
        
        return uniqueList;
    }
}

const list1 = [1, 1, 1, 2, 3, 4, 4, 5];
console.log(Solution.removeDuplicates(list1));
